src/components/Complete/index.js

```diff
@@ -1,33 +1,16 @@
 import { h, Component } from 'preact'
-import theme from '../Theme/style.css'
-import style from './style.css'
-import { trackComponent } from '../../Tracker'
 
-class Complete extends Component {
-  constructor (props) {
-    super(props)
-  }
-
-  componentDidMount () {
-    this.props.nextStep()
-  }
+import MobileSuccess from '../crossDevice/MobileSuccess'
+import VerificationComplete from './VerificationComplete.js'
 
-  render ({message, submessage}) {
+class Complete extends Component {
+  render (props) {
     return (
-      <div>
-      <div className={theme.step}>
-        <span className={`${theme.icon}  ${style.icon}`}></span>
-        <h1 className={`${theme.title} ${theme.center}`}>{message}</h1>
-        <p className={`${theme["mbottom-large"]} ${theme.center} ${style.submessage}`}>{submessage}</p>
-      </div>
-    </div>
+      this.props.mobileFlow ?
+        <MobileSuccess {...props}/> :
```

Comment:
I think this screen will only show if the `complete` step is specified. I tried to test it and there's a bug passing the steps so I will try to fix that. HoweverI suspect this needs to change so we always see MobileSuccess if using  a cross device client

Reply:
Yes, I literally just thought about that while doing a refactoring. I will fix the display on `MobileSuccess`

Reply:
I've pushed a change so that we can always display the cross device ClientSuccess (old MobileSuccess), even when complete is not present. They are two different steps now.